pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -211,16 +215,17 @@ let
     version = "8.0";
     nativeBuildInputs = [ ];
     propagatedBuildInputs = [ compat ];
+    buildInputs = [ stdenv.cc ];
```

Comment:
I don't think this is needed?

Reply:
That’s like _THE MOST ESSENTIAL PART_  of all it. It makes sure the cc-wrapper’s setup-hook runs and makes buildBuild deps (in this case zlib) available for the buildBuild compiler.

Reply:
Oh, sorry, I was thinking about a different line. Let me think about this one.